tensorflow/lite/CMakeLists.txt

```diff
@@ -201,6 +201,9 @@ if(CMAKE_SYSTEM_NAME MATCHES "Android")
 endif()
 # Build a list of source files to compile into the TF Lite library.
 populate_tflite_source_vars("." TFLITE_SRCS)
+if(CMAKE_SYSTEM_NAME MATCHES "Windows" AND BUILD_SHARED_LIBS)
+  list(FILTER TFLITE_SRCS EXCLUDE REGEX ".*simple_memory_arena_debug_dump\\.cc$")
```

Comment:
I think we can actually filter out `simple_memory_arena.cc` and `simple_memory_arena_debug_dump.cc` always. Could you update the PR?

Reply:
When I do that, linking to the library fails with the following error:

```
/nix/store/039g378vc3pc3dvi9dzdlrd0i4q93qwf-binutils-2.39/bin/ld: /home/talyz/.conan/data/tensorflow-lite/2.10.0/_/_/package/a72dbe6c14f82d8449af2c5282a7ce7cd68535a9/lib/libtensorflow-lite.so: undefined reference to `tflite::SimpleMemoryArena::Commit(TfLiteContext*)'
/nix/store/039g378vc3pc3dvi9dzdlrd0i4q93qwf-binutils-2.39/bin/ld: /home/talyz/.conan/data/tensorflow-lite/2.10.0/_/_/package/a72dbe6c14f82d8449af2c5282a7ce7cd68535a9/lib/libtensorflow-lite.so: undefined reference to `tflite::SimpleMemoryArena::ClearPlan()'
/nix/store/039g378vc3pc3dvi9dzdlrd0i4q93qwf-binutils-2.39/bin/ld: /home/talyz/.conan/data/tensorflow-lite/2.10.0/_/_/package/a72dbe6c14f82d8449af2c5282a7ce7cd68535a9/lib/libtensorflow-lite.so: undefined reference to `tflite::SimpleMemoryArena::Deallocate(TfLiteContext*, tflite::ArenaAllocWithUsageInterval const&)'
/nix/store/039g378vc3pc3dvi9dzdlrd0i4q93qwf-binutils-2.39/bin/ld: /home/talyz/.conan/data/tensorflow-lite/2.10.0/_/_/package/a72dbe6c14f82d8449af2c5282a7ce7cd68535a9/lib/libtensorflow-lite.so: undefined reference to `tflite::SimpleMemoryArena::ReleaseBuffer()'
/nix/store/039g378vc3pc3dvi9dzdlrd0i4q93qwf-binutils-2.39/bin/ld: /home/talyz/.conan/data/tensorflow-lite/2.10.0/_/_/package/a72dbe6c14f82d8449af2c5282a7ce7cd68535a9/lib/libtensorflow-lite.so: undefined reference to `tflite::SimpleMemoryArena::Allocate(TfLiteContext*, unsigned long, unsigned long, int, int, int, tflite::ArenaAllocWithUsageInterval*)'
/nix/store/039g378vc3pc3dvi9dzdlrd0i4q93qwf-binutils-2.39/bin/ld: /home/talyz/.conan/data/tensorflow-lite/2.10.0/_/_/package/a72dbe6c14f82d8449af2c5282a7ce7cd68535a9/lib/libtensorflow-lite.so: undefined reference to `tflite::SimpleMemoryArena::ResolveAlloc(TfLiteContext*, tflite::ArenaAllocWithUsageInterval const&, char**)'
/nix/store/039g378vc3pc3dvi9dzdlrd0i4q93qwf-binutils-2.39/bin/ld: /home/talyz/.conan/data/tensorflow-lite/2.10.0/_/_/package/a72dbe6c14f82d8449af2c5282a7ce7cd68535a9/lib/libtensorflow-lite.so: undefined reference to `tflite::SimpleMemoryArena::DumpDebugInfo(std::string const&, std::vector<int, std::allocator<int> > const&) const'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/test_package.dir/build.make:201: test_package] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/test_package.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

I guess it should still be possible to filter out `simple_memory_arena_debug_dump.cc` on all platforms, though.